math/isautomorphic_test.go

```diff
@@ -0,0 +1,61 @@
+// isautomorphic_test.go
```

Comment:
You also don't need to add this unless you want to use this to sign that you contributed here - see our contribution guidelines for how to write such a thing.

Reply:
ok removing the comment